test/test_helper/common.bash

```diff
@@ -75,7 +75,7 @@ function container_is_running() {
 # @param ${1} port
 # @param ${2} container name
 function wait_for_tcp_port_in_container() {
-    repeat_until_success_or_timeout --fatal-test "container_is_running ${2}" "${TEST_TIMEOUT_IN_SECONDS}" docker exec "${2}" /bin/sh -c "nc -z 0.0.0.0 ${1}"
+    repeat_until_success_or_timeout --fatal-test "container_is_running ${2}" "${TEST_TIMEOUT_IN_SECONDS}" docker exec ${2} /bin/sh -c "nc -z 0.0.0.0 ${1}"
```

Comment:
what is this about?

Reply:
I added `""` around `${2}`. Is this a no-go?

Reply:
The diff says the "" were removed.

Reply:
Now as you say it, I added them in the first place, but I wasn't about this change breaking something. So I removed them again. This diff is strange though since we should be back at default.

Reply:
This diff is outdated. The current version uses `""`.